flocker/acceptance/testtools.py

```diff
@@ -157,6 +151,10 @@ def _clean_node(test_case, node):
     # http://doc-dev.clusterhq.com/advanced/cleanup.html#removing-zfs-volumes
```

Comment:
This is no longer a valid anchor link, the heading has been changed to bold text.